transcriptic/cli.py

```diff
@@ -638,13 +638,18 @@ def format_cmd(manifest):
 @click.option(
     "--workcell-id",
     "-w",
-    help="The workcell id to use for the device set. This is not permitted along with the `deviceSet` option.",
+    help="The workcell id to use for the device set. This is not permitted along with the `device-set` or `session-id` option.",
 )
 @click.option(
     "--device-set",
     "-d",
     type=click.File("r"),
-    help="A DeviceSet json file to use for scheduling. This is not permitted along with the `workcellId` option.",
+    help="A DeviceSet json file to use for scheduling. This is not permitted along with the `workcell-id` or `session-id` options.",
+)
+@click.option(
+    "--session-id",
+    "-s",
+    help="A Session id that already exists in your scle test workcell instance. This is not permitted along with the `workcell-id` or `device-set` options.",
```

Comment:
Let's rephrase this slightly to talk about the effect of specifying this. Instead of stating this references a session Id which already exists, let's state what the session id is used for